config/locales/en.yml

```diff
@@ -1859,6 +1859,7 @@ en:
   order_hub_info: Hub Info
   order_back_to_store: Back To Store
   order_back_to_cart: Back To Cart
+  order_back_to_website: Back To Website
```

Comment:
I reckon that you could have squashed all these commits in one. They don't make sense on their own.